src/main/java/racing/Racing.java

```diff
@@ -12,6 +12,16 @@ public Racing(int cars, int round) {
         this.round = round;
     }
 
+    public Racing(String[] cars, int round) {
```

Comment:
```suggestion
    public Racing(String[] carNames, int round) {
```

문맥상 유추하자면 차의 이름이 더 적절한 매개변수명으로 보입니다.